types/lodash/common/common.d.ts

```diff
@@ -15,12 +19,13 @@ declare module "../index" {
         T extends object | null | undefined ? Object<T> :
         Primitive<T>;
     type ExpChain<T> =
-        T extends { __trapAny: any } ? CollectionChain<any> & FunctionChain<any> & ObjectChain<any> & PrimitiveChain<any> & StringChain :
-        T extends null | undefined ? never :
-        T extends string ? StringChain<T> :
-        T extends (...args: any) => any ? FunctionChain<T> :
-        T extends List<infer U> | null | undefined ? CollectionChain<U> :
-        T extends object | null | undefined ? ObjectChain<T> :
+        [T] extends [{ __trapAny: any }] ? CollectionChain<any> & FunctionChain<any> & ObjectChain<any> & PrimitiveChain<any> & StringChain :
+        NullOrUndefinedOnly<T> extends true ? never :
```

Comment:
For pure null/undefined, maybe it's OK to use `PrimitiveChain<T>`? I suspect these were filtered out before under the assumption that null/undefined would always be part of a union with other types.